hdate/tekufot.py

```diff
@@ -0,0 +1,265 @@
+"""
+The class attempts to compute:
+    - The tekufot (seasons) in the Hebrew calendar: Nissan, Tammuz, Tishrei, and Tevet.
+    - Calculaltion are based on Shmuel calendar (Talmud Bavli, Eruvin 56a)
+    - Rav Ada’s Tekufah: Closer to the astronomical solar year,
+      shorter by 5min than Schmuel (365 days, 5 hours, 997 parts/chalakim) is not used
+    - Cheilat Geshamim start date, which differs between the diaspora and Israel.
+    - Halachic prayer periods based on key Jewish holidays and seasonal changes.
+    - Appropriate prayer phrases depending on the current date, tradition, and language.
+"""
+
+import datetime
```

Comment:
Please import datetime as dit
Keeps it consistent with the rest of the library.

Reply:
ok